src/App.js

```diff
@@ -1,24 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] The `App.js` component should be class-based. In this milestone, the objective is to practice the class-based React component.